disperser/api/proto/encoder/v2/encoder.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+option go_package = "github.com/Layr-Labs/eigenda/api/grpc/encoder/v2";
+package encoder.v2;
+
+service Encoder {
+  rpc EncodeBlobToChunkStore(EncodeBlobToChunkStoreRequest) returns (EncodeBlobToChunkStoreReply) {}
```

Comment:
Describe what does this API do?

Also should it be just `EncodeBlob`? Is storing the results to chunk store an important thing to call out in the name?

Reply:
Changed it to `EncodeBlob`